docs/introduction.rst

```diff
@@ -119,42 +129,49 @@ Using the previous 2H **Site** object example, we construct a simple single-site
 **SpinSystem** object, as shown below.
 
 .. _listing_uncoupled_spin_system:
-.. code-block:: json
-  :linenos:
-  :caption: An example of uncoupled 2H spin system in JSON representation.
-
-  {
-      "name": "2H uncoupled spin system",
-      "description": "An optional description of the spin system",
-      "sites": [
-        {
-          "isotope": "2H",
-          "isotropic_chemical_shift": "4.1 ppm",
-          "shielding_symmetric": {
-              "zeta": "12.12 ppm",
-              "eta": 0.82,
-              "alpha": "5.45 rad",
-              "beta": "4.82 rad",
-              "gamma": "0.5 rad"
-          },
-          "quadrupolar": {
-              "Cq": "1.47 MHz",
-              "eta": 0.27,
-              "alpha": "0.212 rad",
-              "beta": "1.231 rad",
-              "gamma": "3.1415 rad"
-          }
-        }
-      ],
-      "abundance": "0.148%"
-  }
+.. plot::
+    :format: doctest
+    :context: close-figs
```

Comment:
There is no need for a `plot` directive. There are no Matplotlib figures in these examples. Just use a `doctest` directive.

Reply:
Will be done in #98